websites/B/Brain.fm/presence.ts

```diff
@@ -0,0 +1,111 @@
+		else if (document.location.pathname.startsWith("/forgot-password"))
+			presenceData.details = "Resetting Password...";
+	} else if (window.location.hostname.startsWith("my")) {
+		if (document.location.pathname === "/") {
+			if (
+				document.querySelector(
+					"#root > div > div.sc-gKsewC.sc-iBPRYJ.sc-gGiJkG.efCouF.cOWINi.jzdkxv"
+				)
+			)
+				presenceData.details = "Logging into Brain.fm";
+			else if (
+				document.querySelector("#root > div > div > div > div.sc-jsNsAp.byZiWf")
+			)
+				presenceData.details = "Choosing Mode...";
+		} else if (document.location.pathname.startsWith("/signup"))
+			presenceData.details = "Signing Up for Brain.fm";
+		else if (document.location.pathname.startsWith("/welcome"))
+			presenceData.details = "Getting started with Brain.fm";
+		else if (
+			["/focus", "/relax", "/sleep"].includes(document.location.pathname)
+		) {
+			switch (document.location.pathname) {
+				case "/focus":
+					presenceData.state = `Focusing • ${document
+						.querySelector(
+							"#root > div > div > div.sc-sCrnh.fGBZrg > div.sc-gJLCiq.iGquue > div.sc-kBizdN.gGaVww > div.sc-jNpQCG.dAEozB > div > div > span"
+						)
+						.textContent.trim()}`;
+					break;
+				case "/relax":
+					presenceData.state = `Relaxing • ${document
+						.querySelector(
+							"#root > div > div > div.sc-sCrnh.fGBZrg > div.sc-gJLCiq.iGquue > div.sc-kBizdN.gGaVww > div.sc-jNpQCG.dAEozB > div > div > span"
+						)
+						.textContent.trim()}`;
+					break;
+				case "/sleep":
+					presenceData.state = `Sleeping • ${document
+						.querySelector(
+							"#root > div > div > div.sc-sCrnh.fGBZrg > div.sc-gJLCiq.iGquue > div.sc-kBizdN.gGaVww > div.sc-jNpQCG.dAEozB > div > div > span"
+						)
+						.textContent.trim()}`;
+					break;
+			}
+
+			const songTypeInfo = document
+				.querySelector(
+					"#root > div > div > div.sc-VhGJa.kMMYFW > div.sc-jwBoPJ.glJHPW > div > div.sc-chbBtj.ciMhIb > div > div > div > div > div.sc-leFDRL.iaABgi"
+				)
+				.textContent.trim()
+				.split(" • ");
+
+			presenceData.details = `${document
+				.querySelector(
+					"#root > div > div > div.sc-VhGJa.kMMYFW > div.sc-jwBoPJ.glJHPW > div > div.sc-chbBtj.ciMhIb > div > div > div > div > div.sc-hDjjHo.jwdsSo"
+				)
+				.textContent.trim()} • ${songTypeInfo[0]}`;
+
+			presenceData.largeImageKey = document
+				.querySelector(
+					"#root > div > div > div.sc-VhGJa.kMMYFW > div.sc-jwBoPJ.glJHPW > div > div.sc-chbBtj.ciMhIb > div > div > div > img"
+				)
+				.getAttribute("src");
+			presenceData.smallImageKey = "logo";
+			presenceData.smallImageText = songTypeInfo[1];
+		} else if (document.location.pathname.startsWith("/user"))
+			presenceData.details = "Viewing user information...";
+		else if (document.location.pathname.startsWith("/payment"))
+			presenceData.details = "Viewing payment information...";
+		else if (document.location.pathname.startsWith("/forgot-password"))
+			presenceData.details = "Resetting Password...";
```

Comment:
Would this condition not also be met on L37

Reply:
My mistake! Resolved 👍🏻